assets/css/_announcements-interests-nav.scss

```diff
@@ -36,14 +40,17 @@
   }
 }
 
-.mine.selected {
-  + .select-interests-link,
-  .caret {
-    border-bottom: $dark-blue 1px solid;
+.interests-message {
+  border: $base-border;
+  text-align: center;
+  transform: translateY(-1px);
```

Comment:
Reason for this? Weird alignment things? Just curious.

Reply:
Your comment made me realize I don't need this. I was using it to cancel out the border top on the container. I should really just have a border bottom and kill this declaration instead.

Reply:
@enatario https://github.com/thoughtbot/constable/pull/457/commits/3220c7c8a65175f974937ba2e8e18c6f30e8ed4e